index/server/entrypoint.sh

```diff
@@ -10,7 +11,27 @@ if [ ! -e "$DEVFILE_INDEX" ]; then
     exit 1
 fi
 
+# Start the registry viewer
+npm start &
+
+# Wait for server to start
+until $(curl --output /dev/null --silent --head --fail http://localhost:3000/viewer); do
+    printf '.'
+    sleep 1
+done
+
+# Copy over the registry viewer config
+# cp -rf /viewer-config/devfile-registry-hosts.json /app/config/
```

Comment:
how do we set the local path using config in this case

Reply:
This should be removed. The config is mounted directly to that folder as a volume via config map.

Reply:
Fixed.